src/audio/module_adapter/module/cadence.c

```diff
@@ -113,6 +113,49 @@ static int cadence_code_get_api_id(uint32_t compress_id)
 	}
 }
 
+#if CONFIG_IPC_MAJOR_4
```

Comment:
to avoid this occur multiple times in one source file, normally use:

#if CONFIG_IPC_MAJOR_3
....
#elif #if CONFIG_IPC_MAJOR_4
....
#else
#error Unknown IPC major version
#endif